src/events/_event.html

```diff
@@ -61,39 +64,39 @@ <h1 class="post_heading">
     {% endif %}
     {% from "_event-post-macros.html" import event_venue as event_venue %}
-    {{ event_venue(event) }}
+    {{ event_venue(event, event_state) }}
     </header>
     <div class="post_body">
-        {% if (request.url_rule.endpoint != 'event_archive') %}
-            <aside class="post_inset post_inset__right line-container event-status">
-                <div class="line-container_body">
-                    <h1 class="u-visually-hidden">Event reservation and viewing details</h1>
-                    <div class="event-status_rsvp">
-                      <p>
-                          <strong>This event requires an RSVP.</strong>
-                      </p>
-                      <p>
-                          <a href="mailto:reserve@cfpb.gov?subject=Event RSVP&body=To RSVP, please fill in your first and last name below and send this email.%0D%0A%0D%0AFirst name:%0D%0ALast name:" class="btn"><span class="btn_icon__left cf-icon cf-icon-email-social"></span> reserve@cfpb.gov</a>
-                      </p>
-                    </div>
-                    {% if (event.live_stream) %}
-                    <div class="event-status_livestream">
-                      <p>
-                          {# TODO Change to livestream icon when that's added to cf-icons. #}
-                          <span class="cf-icon cf-icon-livestream"></span>
-                          <strong>Available on Live stream.</strong>
-                      </p>
-                      <p class="event-meta">
-                          <span class="event-meta_description event-meta_description__block">
-                              Video link available:
-                          </span>
-                          {% from "_event-post-macros.html" import event_meta_datetime as event_meta_datetime %}
-                          {{ event_meta_datetime(event.live_stream.date) }}
-                      </p>
-                    </div>
-                    {% endif %}
+        {% if (event_state != 'past') %}
+        <aside class="post_inset post_inset__right line-container event-status">
+            <div class="line-container_body">
+                <h1 class="u-visually-hidden">Event reservation and viewing details</h1>
+                <div class="event-status_rsvp">
+                  <p>
+                      <strong>This event requires an RSVP.</strong>
+                  </p>
+                  <p>
+                      <a href="mailto:reserve@cfpb.gov?subject=Event RSVP&body=To RSVP, please fill in your first and last name below and send this email.%0D%0A%0D%0AFirst name:%0D%0ALast name:" class="btn"><span class="btn_icon__left cf-icon cf-icon-email-social"></span> reserve@cfpb.gov</a>
+                  </p>
+                </div>
+                {% if (event.live_stream) %}
+                <div class="event-status_livestream">
+                  <p>
+                      {# TODO Change to livestream icon when that's added to cf-icons. #}
```

Comment:
livestream icon is in cf-icons now https://github.com/cfpb/cf-icons/blob/gh-pages/demo/index.html#L703